src/clients/bridges/CrossChainTransferClient.ts

```diff
@@ -89,14 +89,12 @@ export class CrossChainTransferClient {
     this.log("Updating cross chain transfers", { monitoredChains });
 
     const outstandingTransfersPerChain = await Promise.all(
-      monitoredChains.map((chainId) =>
-        this.adapterManager.getOutstandingCrossChainTokenTransferAmount(chainId, l1Tokens)
-      )
+      monitoredChains.map(async (chainId) => [
+        chainId,
+        await this.adapterManager.getOutstandingCrossChainTokenTransferAmount(chainId, l1Tokens),
+      ])
```

Comment:
Minor: Since this is bundled into a `Promise.all()` is it still required to make the closure async?

```suggestion
      monitoredChains.map((chainId) => [
        chainId,
        this.adapterManager.getOutstandingCrossChainTokenTransferAmount(chainId, l1Tokens),
      ])
```

Reply:
It's required because we're not returning a promise of `Event[]` - we're returning an array of type: `Promise<[number, Event[]]>`. If we didn't await we'd have a return type of `Promise<[number, Promise<Event[]>]>`